packages/nouns-contracts/contracts/client-incentives/Rewards.sol

```diff
@@ -165,20 +185,40 @@ contract Rewards is NounsClientToken, UUPSUpgradeable, PausableUpgradeable {
      * @notice Register a client, mints an NFT and assigns a clientId
      * @return uint32 the newly assigned clientId
      */
-    function registerClient(
-        string calldata name,
-        string calldata description
-    ) public override whenNotPaused returns (uint32) {
+    function registerClient(string calldata name, string calldata description) external whenNotPaused returns (uint32) {
         RewardsStorage storage $ = _getRewardsStorage();
 
-        uint32 tokenId = super.registerClient(name, description);
+        uint32 tokenId = $.nextTokenId;
+        $.nextTokenId++;
```

Comment:
@eladmallel nit: maybe more gas efficient to do `$.nextTokenId = tokenId + 1` here

Reply:
fixed